### PR TITLE
frontend: Fix small UI regressions

### LINKED ISSUE #2049: Some UI regressions
Here are some UI regressions I've noticed

- [ ] Menu labels are uppercase
image

- [ ] Logs view has large padding
image

### PR DESCRIPTION
**Pull request by sniok**
Small regressions in SimpleTable, LogViewer, Terminal, ShowHideLabel, SidebarItem

fixes #2049

### CODE REVIEW

File: frontend/src/components/common/LogViewer.tsx [hunk 1]
```diff
@@ -148,7 +148,7 @@ export function LogViewer(props: LogViewerProps) {
             width: '100%',
             height: '100%',
             '& .terminal.xterm': {
-              padding: 10,
+              padding: '10px',
```
Comment (joaquimrocha) [inline frontend/src/components/common/LogViewer.tsx:151]: I think we should avoid pixel units for padding/spacing but instead use the `theme.spacing()` function. Can you try with `theme.spacing(1)`?

File: frontend/src/components/common/ShowHideLabel/ShowHideLabel.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default function ShowHideLabel(props: ShowHideLabelProps) {
     <Box display={expanded ? 'block' : 'flex'}>
       <label
         id={labelIdOrRandom}
-        style={{ wordBreak: 'break-all', whiteSpace: 'normal' }}
+        style={{ wordBreak: 'normal', whiteSpace: 'normal' }}
```
Comment (joaquimrocha) [inline frontend/src/components/common/ShowHideLabel/ShowHideLabel.tsx:46]: Is the break-all change actually causing a regression? (Do you have a screenshot?)

File: frontend/src/components/common/Terminal.tsx [hunk 3]
```diff
@@ -393,7 +393,7 @@ export default function Terminal(props: TerminalProps) {
             overflow: 'hidden',
             width: '100%',
             '& .terminal.xterm': {
-              padding: 10,
+              padding: '10px',
```
Comment (joaquimrocha) [inline frontend/src/components/common/Terminal.tsx:396]: use `theme.spacing()`

File: frontend/src/components/common/ShowHideLabel/ShowHideLabel.tsx (hunk 2 above)
Comment (sniok) [reply frontend/src/components/common/ShowHideLabel/ShowHideLabel.tsx]: yeah you're right it's not, I've removed it. It was just really hard to read error messages in the cluster overview and I thought it must be a regression. We can come back to this later

**Review by joaquimrocha — APPROVED** [review]

**Pull request merged**